targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_MODULE_UBLOX_ODIN_W2/TARGET_UBLOX_EVK_ODIN_W2/hal_overrides.c

```diff
@@ -0,0 +1,24 @@
+#include "stm32f4xx_hal.h"
```

Comment:
Can you please add a license header here ?

Reply:
Fixed